recipes/mosdef_cassandra/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set name = "mosdef_cassandra" %}
+{% set version = "0.1.0" %}
+{% set sha256 = "88e87ee6cbc266e43d169e433e010789b8ccdc1552f046afd94db8f9c75ed771" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/MaginnGroup/mosdef_cassandra/archive/v{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: True  # [win]
+  skip: True  # [py<35]
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
```

Comment:
This doesn't quite match the requirements listed here: https://github.com/MaginnGroup/mosdef_cassandra/blob/master/requirements.txt?